web/src/components/legend/Co2LegendRounded.tsx

```diff
@@ -0,0 +1,56 @@
+import { extent } from 'd3-array';
+import { ScaleLinear, scaleLinear } from 'd3-scale';
+import { TFunction } from 'i18next';
+import { CarbonUnits } from 'utils/units';
+
+import { spreadOverDomain } from './utils';
+
+function Co2LegendRounded({
```

Comment:
I put this legend in components/legends because it shared some functionality with ColorBar. Do you think I should put all legends in components/legends or should I leave the legends that are only used in the BarBreakdown in the bar-breakdown directory?

Reply:
I think leaving it is fine for now and then on the next sync we can discuss where to put all the components and such. I'm having a similar question/discussion about where to put all the specific buttons in #6867